packages/twenty-server/src/workspace/workspace-migration-runner/workspace-migration-runner.service.ts

```diff
@@ -140,6 +140,11 @@ export class WorkspaceMigrationRunnerService {
       }),
       true,
     );
+
+    // Enable totalCount for the table
+    await queryRunner.query(`
```

Comment:
I've tested locally to create a new custom object and it didn't add the comment on the table 🤔 @magrinj

Reply:
Hmm strange, I've also tested it locally and it works properly, did you refresh your table view ?

I just test it again, it works properly on my side:
<img width="952" alt="Screenshot 2024-01-12 at 8 51 46 AM" src="https://github.com/twentyhq/twenty/assets/3551795/cfcb408e-d323-4ac1-acf5-bd807ea9e766">

Reply:
<img width="373" alt="Screenshot 2024-01-12 at 10 16 32" src="https://github.com/twentyhq/twenty/assets/1834158/b1b94ec6-306a-475d-ac04-c18dcb8ce9be">
<img width="410" alt="Screenshot 2024-01-12 at 10 16 38" src="https://github.com/twentyhq/twenty/assets/1834158/8a9f0d56-abd9-44c4-8314-a1ab0d3315f4">

Reply:
My bad! All good!